Prox/Prox/PlaceDetails/FilterViewController.swift

```diff
@@ -38,13 +38,21 @@ class FilterViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+    private func setupShadow(container: UIView) {
+        container.layer.masksToBounds = false
```

Comment:
For future reference, in my next PR, I defined the shadow qualities in `Style.shadow*`